meta/main.yml

```diff
@@ -0,0 +1,57 @@
+galaxy_info:
+  author: your name
```

Comment:
Update this?

Reply:
Ack! I will delete this for now, will add it when I'm preparing the role for galaxy.